### ISSUE TITLE
Small suggestion: Form fields

### ISSUE THREAD

**Issue by mediaformat** [kevquirk/simple.css-site#7]
For accessibility radio inputs should be wrapped in labels (allows users to click on the text vs limiting it to the radio button)
```
<p>
  <label>Enquiry type:</label><br>
  <label><input checked="checked" name="type" type="radio" value="sales"> Sales </label><br>
  <label><input name="type" type="radio" value="support"> Support </label><br>
  <label><input name="type" type="radio" value="billing"> Billing </label>
</p>
```
